spec/features/users/show_spec.rb

```diff
@@ -1,39 +1,45 @@
 # frozen_string_literal: true
 
-require 'rails_helper'
+require "rails_helper"
 
-describe 'Users#show' do
+describe "Users" do
```

Comment:
I actually want the action name. It indicates which page the feature spec starts on.

Reply:
Did Rubocop complain about it? If so, we can drop it.

Reply:
It did:

```RSpec/DescribeClass: The first argument to describe should be the class or module being tested.```